src/figuresClasses.ts

```diff
@@ -1,11 +1,95 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string;
+
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(color: string, a: number, b: number, c: number) {
+    this.shape = 'triangle';
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
```

Comment:
[CODE KNOWLEDGE] - You don't need to manually assign constructor parameters to class properties when using access modifiers. TypeScript does this automatically for you. Consider using access modifiers (like public or private) in your constructor parameters to simplify your code.